src/ch05-02-example-structs.md

```diff
@@ -1,71 +1,72 @@
-## An Example Program Using Structs
+## Przykładowy program wykorzystujący struktury
 
-To understand when we might want to use structs, let’s write a program that
-calculates the area of a rectangle. We’ll start with single variables, and then
-refactor the program until we’re using structs instead.
+Aby pokazać, kiedy warto skorzystać ze struktur, napiszmy program, który
+policzy pole prostokąta. Zaczniemy od pojedynczych zmiennych, potem zrefaktorujemy
+program, tak aby używał struktur.
 
-Let’s make a new binary project with Cargo called *rectangles* that will take
-the width and height of a rectangle specified in pixels and calculate the area
-of the rectangle. Listing 5-8 shows a short program with one way of doing
-exactly that in our project’s *src/main.rs*.
+Stwórzmy projekt aplikacji binarnej przy użyciu Cargo. Nazwijmy go *prostokąty*.
+Jako wejście przyjmie szerokość i wysokość danego prostokąta i wyliczy
+jego pole. Listing 5-8 pokazuje krótki program obrazujący jeden ze sposobów,
+w jaki możemy to wykonać.
 
-<span class="filename">Filename: src/main.rs</span>
+<span class="filename">Plik: src/main.rs</span>
 
 ```rust
 fn main() {
     let width1 = 30;
     let height1 = 50;
 
     println!(
-        "The area of the rectangle is {} square pixels.",
+        "Pole prostokąta wynosi {} pikseli kwadratowych."
         area(width1, height1)
     );
 }
 
-fn area(width: u32, height: u32) -> u32 {
-    width * height
+fn area(height: u32, width: u32) -> u32 {
+    height * width
 }
 ```
 
-<span class="caption">Listing 5-8: Calculating the area of a rectangle
-specified by separate width and height variables</span>
+<span class="caption">Listing 5-8: Kalkulacja pola prostokąta określonego
+poprzez oddzielne zmienne wymiarów: szerokości i wysokości</span>
 
-Now, run this program using `cargo run`:
+Uruchommy program komendą `cargo run`:
 
 ```text
-The area of the rectangle is 1500 square pixels.
+Pole prostokąta wynosi 1500 pikseli kwadratowych.
 ```
 
-Even though Listing 5-8 works and figures out the area of the rectangle by
-calling the `area` function with each dimension, we can do better. The width
-and the height are related to each other because together they describe one
-rectangle.
+Mimo że w Listingu 5-8 wszystko wygląda OK, a więc wylicza pole prostokąta
```

Comment:
Proponuję:
Pomimo że program z Listingu 5-8 wygląda dobrze i poprawnie oblicza pole prostokąta wywołując funkcję `area` podając oba wymiary, to możemy go napisać lepiej.

Reply:
czemu `pomimo`, a nie `mimo` na początku zdania?

Reply:
Może być "mimo", to tylko kwestia przyzwyczajeń.

Reply:
a ok, myślałem że dostanę link do sjp.pl bo już dwóch rzeczy się od Ciebie od nich nauczyłem;

*mimo* wszystko od 3.1 jest już `pomimo`